#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("min_height", double_t, 0, "the minimum height of the prism craeted by the planes", 0.0, -5.0, 5.0)
gen.add("max_height", double_t, 0, "the minimum height of the prism craeted by the planes", 0.5, -5.0, 5.0)
gen.add("maginify", double_t, 0, "maginify polygon region (m)", 0.0, -1.0, 1.0)
gen.add("keep_organized", bool_t, 0, "keep organized pointcloud or not", True)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "MultiPlaneExtraction"))
